pkg/cloud/cloud.go

```diff
@@ -653,7 +656,11 @@ func (c *cloud) ResizeOrModifyDisk(ctx context.Context, volumeID string, newSize
 
 	response, err := c.ec2.ModifyVolumeWithContext(ctx, req)
 	if err != nil {
-		return 0, fmt.Errorf("unable to modify AWS volume %q: %w", volumeID, err)
+		if isAWSErrorModifyVolumeInvalidParameter(err) {
+			// Wrap the many possible EC2 errors with cloud error, but pass on actual EC2 error to operator
```

Comment:
I thought this is the only way to preserve the original error message from AWS explaining why this was an invalid argument. Am I mistaken?

Reply:
You're right - that's a good reason to do so. Can we add that reasoning (preserving the error message) to the comment because it wasn't obvious to me?

Reply:
How would you phrase it better than what I already wrote: `// Wrap the many possible EC2 errors with cloud error, but pass on actual EC2 error to operator`

Would: "Wrap error in order to preserve original error message from AWS why this was an invalid argument" be better?

Reply:
Changed comment to `// Wrap error to preserve original message from AWS as to why this was an invalid argument` so it's clearer. Thanks for this feedback. 

Resolving.